lib/fluent/log.rb

```diff
@@ -51,6 +51,8 @@ module TTYColor
     LOG_TYPES = [LOG_TYPE_SUPERVISOR, LOG_TYPE_WORKER0, LOG_TYPE_DEFAULT].freeze
     LOG_ROTATE_AGE = %w(daily weekly monthly)
 
+    IGNORE_SAME_LOG_MAX_CACHE_SIZE = 100 # If need, make this an option of system config.
```

Comment:
It might be better to make it larger value for safety, e.g. `1000`.
I think there is no impact to memory & CPU usage at this level.

Reply:
Thanks for your review!
I see! I will change it.

Reply:
DONE!